main.js

```diff
@@ -440,7 +439,7 @@ app.get('/profiles', async function (req, res) {
 
 	// TODO: error handling
 	// TODO: really, we should do pagination for this kind of request.
-	const usersCursor = db.collection('BBY-6_users').find()
+	const usersCursor = db.collection('BBY-6_users').find({'admin': true})
```

Comment:
Should document that this only gets admin users, route is still called `'/profiles'`.

Reply:
wrote a short comment that this gets admin profiles, may update name later (or not?)